### PR TITLE
docs: update lazy-load-translations.md

### PR DESCRIPTION
**Pull request by defaye**
The following changes are required for lazy loading to work properly.

### CODE REVIEW

File: docs/content/en/lazy-load-translations.md [hunk 1]
```diff
@@ -46,15 +46,17 @@ Configuration example:
   ],
   lazy: true,
   langDir: 'lang/',
-  defaultLocale: 'en'
+  vueI18n: {
+    defaultLocale: 'en'
+  }
```
Comment (rchl) [inline docs/content/en/lazy-load-translations.md]: This is not correct. `vue-i18n` doesn't support a `defaultLocale` option.

**Comment by rchl** [comment]
Whatever changes are done in `en` have to be duplicated in `es` documentation also.

File: docs/content/en/lazy-load-translations.md (hunk 1 above)
Comment (defaye) [reply docs/content/en/lazy-load-translations.md]: You're right. Sorry. This was meant to be a reference to `fallbackLocale`. I think that without it, it wasn't working for me when using lazy-loading.

Comment (defaye) [reply docs/content/en/lazy-load-translations.md]: Actually, I think it was working fine.  It seems that something else was at play, as some kind of caching possibly lead me to believe this was the critical change, but I've left the changes for the missing return statement.

**Pull request closed without merging**